frame/instrumental-strategy-pablo/src/mock/runtime.rs

```diff
@@ -251,6 +251,45 @@ impl pallet_pablo::Config for MockRuntime {
 	type WeightInfo = ();
 }
 
+// -------------------------------------------------------------------------------------------------
+//                                       Instrumental Strategy
+// -------------------------------------------------------------------------------------------------
+
+parameter_types! {
+	pub const InstrumentalStrategyPalletId: PalletId = PalletId(*b"dynamic_");
+}
+
+impl instrumental_strategy::Config for MockRuntime {
+	type Event = Event;
+	type WeightInfo = ();
+	type AssetId = CurrencyId;
+	type Balance = Balance;
+	type VaultId = VaultId;
+	type Vault = Vault;
+	type PabloStrategy = PabloStrategy;
+	type MaxAssociatedVaults = MaxAssociatedVaults;
+	type PalletId = InstrumentalStrategyPalletId;
+}
+
+// -------------------------------------------------------------------------------------------------
+//                                           Instrumental
+// -------------------------------------------------------------------------------------------------
+
+parameter_types! {
+	pub const InstrumentalPalletId: PalletId = PalletId(*b"strm____");
```

Comment:
The same question about `b"strm____"`

Reply:
See [the comment above](https://github.com/ComposableFi/composable/pull/1195#discussion_r904899798) (but `instumental`'s mock runtime used here).